app/controllers/api/v1/palettes_controller.rb

```
@@ -8,14 +10,16 @@ def show
     render json: Palette.find(params[:id])
   end
 
-
   def create
-    palette = Palette.create(palette_params)
+    palette = Palette.new(palette_params)
+    palette.user = current_user
+    palette.save
+    render json: palette
```

You need a conditional for if this does not save properly. It should return an error via json if so